.gitignore

```diff
@@ -51,3 +51,4 @@ build.zip
 /coverage
 
 
+.vercel
```

Comment:
We have a private deployment on Vercel for our convenience, making internal testing easier.
So, as discussed earlier rmv this line.